apollo/lib/schemas/user.py

```diff
@@ -1,17 +1,25 @@
 import uuid
 
-from pydantic import validator, constr, BaseModel
+from pydantic import validator, constr, BaseModel, root_validator
 from typing import Optional
 
 from apollo.lib.decorators import with_db_session
+from apollo.lib.exceptions import PydanticValidationError
 from apollo.lib.schemas import ORMBase
 from apollo.lib.schemas.role import RoleSchema
 from apollo.models.user import get_user_by_username
 
 
 def check_for_whitespace(value):
     if ' ' in value:
-        raise ValueError("password can't contain whitespaces")
+        raise ValueError("field can't contain whitespaces")
```

Comment:
```suggestion
        raise ValueError(f"{field} can't contain whitespaces")
```

This way we can keep our clarifying message.